index.html

```diff
@@ -22,9 +22,20 @@ <h1 id="play">let's play</h1>
         </div>
       </div>
     </div>
-    <div id="wrapper">
+    <div id="title-wrapper">
```

Comment:
Po mergu upewnij się proszę że to nie psuje innych styli, dopisanych w międzyczasie